mods/gen4/formats-data.js

```diff
@@ -2075,7 +2075,7 @@ exports.BattleFormatsData = {
 		requiredItem: "Dread Plate",
 	},
 	arceusdragon: {
-		randomBattleMoves: ["swordsdance", "dragonclaw", "earthquake", "recover", "overheat"],
+		randomBattleMoves: ["swordsdance", "dragonclaw", "earthquake", "recover", "calmmind", "judgment", "fireblast"],
```

Comment:
This won't end well.

Reply:
Hmm, yeah, I guess there is some potential nastiness here. I'll revert this.